init/dokku-daemon.service

```diff
@@ -9,12 +9,7 @@ Environment="GROUP=dokku"
 Environment="PERMS=0777"
 EnvironmentFile=-/etc/systemd/system/dokku-daemon.env
 
-ExecStartPre=/bin/bash -c 'DOKKU_LOCK_DIR=$(dirname ${DOKKU_LOCK_PATH}); if [ "$${DOKKU_LOCK_DIR}" != "/tmp" ]; then mkdir -p -m ${PERMS} $${DOKKU_LOCK_DIR}; chown ${USER}:${GROUP} $${DOKKU_LOCK_DIR}; fi'
-ExecStartPre=/bin/bash -c 'DOKKU_SOCK_DIR=$(dirname ${DOKKU_SOCK_PATH}); if [ "$${DOKKU_SOCK_DIR}" != "/tmp" ]; then mkdir -p -m ${PERMS} $${DOKKU_SOCK_DIR}; chown ${USER}:${GROUP} $${DOKKU_SOCK_DIR}; fi'
```

Comment:
Why these removals?

Reply:
I had to make many changes and pull requests recently related to dokku and letsencrypt both, so I cannot remember the reasoning behind this change.

I will revert it and test to verify functionality.